Index: PerformanceAnalytics/R/checkData.R
===================================================================
--- PerformanceAnalytics/R/checkData.R	(revision 1755)
+++ PerformanceAnalytics/R/checkData.R	(working copy)
@@ -67,7 +67,7 @@
                 }
                 else
                     stop("The data cannot be converted into a time series.  If you are trying to pass in names from a data object with one column, you should use the form \'data[rows, columns, drop = FALSE]\'.  Rownames should have standard date formats, such as '1985-03-15'. ")
-            else x = as.xts(x)
+            else x = try.xts(x)
         }
     ) # end switch
 
@@ -109,4 +109,4 @@
 # for full details see the file COPYING
 #
 # $Id$
-###############################################################################
\ No newline at end of file
+###############################################################################
Index: PerformanceAnalytics/R/Drawdowns.R
===================================================================
--- PerformanceAnalytics/R/Drawdowns.R	(revision 1755)
+++ PerformanceAnalytics/R/Drawdowns.R	(working copy)
@@ -46,4 +46,4 @@
 #
 # $Id$
 #
-###############################################################################
\ No newline at end of file
+###############################################################################
